### DISCUSSION TITLE
SSL ERROR TOO LONG HTTPS

### DISCUSSION THREAD

**Discussion by Elionis** [rustdesk/rustdesk-server-pro#200] [General]
### Bug Description

Hi, 
When i turn on SSL with Nginx and use CERTBOT i have this error SSL_ERROR_RX_RECORD_TOO_LONG
On fresh install
Try on Ubuntu and Centos8
Url use : https://domain:21114 (web console)
Nota : on http not problem.

### How to Reproduce

Create an fresh install of rustdesk and put SSL during installation.

### Expected Behavior

After install on web console and using httpS have error : SSL_ERROR_RX_RECORD_TOO_LONG

### Operating system(s) on local side and remote side

CENTOS->8 UBUNTU->22

### RustDesk Version(s) on local side and remote side

1.1.14-6

**Comment by rustdesk** [comment]
This has nothing to do with RustDesk server software, it is pure Nginx SSL setup issue. Or you used wrong url, I guess you meant https://domain, please remove `:21114`.

**Reply by Elionis** [reply]
Hi RustDesk, 
Thanks for reply.
i supposed that the script maken an error in nginx conf when he add in nginx.conf.
In values server {}
Regards

**Comment by LyraHarring** [comment]
As I have also been faced by such an error and was able to find the proper guide at:- https://certera.com/kb/how-to-fix-the-ssl_error_rx_record_too_long-error-in-firefox/. Whether it’s your website or someone else’s, this error message could result from inadequate security on the website you’re attempting to access. In a nutshell, the SSL certificate’s improper configuration is the most typical issue, which prevents your browser from successfully establishing a connection with the server’s port. Hope it helps!
